enes-homework-week1/app.js

```diff
@@ -0,0 +1,163 @@
+{
+  ('use strict');
+  //1.1; 1.2,
+  const listOfBooks = ['ak', 'bak', 'cak', 'dak', 'eak', 'fak', 'gak', 'hak', 'iak', 'jak'];
+
+  //console.log(listOfBooks);
+
+  //1.3
+
+  // const objListOfBooks = Object.assign(listOfBooks);
+  function listMaker() {
+    const root = document.getElementById('root');
+    const h1 = document.createElement('h1');
+    h1.innerText = 'My Book List';
+    root.appendChild(h1);
+    const ul = document.createElement('ul');
+    root.appendChild(ul);
+    listItemMaker();
+    bookListMaker();
+    bookCover();
+  }
+
+  function listItemMaker() {
+    for (let i = 0; i < listOfBooks.length; i++) {
+      const li = document.createElement('li');
+      li.innerText = listOfBooks[i];
+      root.appendChild(li);
```

Comment:
You are appending the `li` to the root but you should append the items to the `ul`. Right now the output if as follows:
```
<ul></ul>
<li>ak</li>
<li>bak</li>
```